src/natcap/invest/urban_mental_health.py

```diff
@@ -0,0 +1,1232 @@
+            kwargs={
+                'ignore_nodata_and_edges': True,
+                'mask_nodata': True,
+                'normalize_kernel': True,
+                'target_datatype': pygeoprocessing.get_raster_info(
+                    file_registry['ndvi_base_aligned_masked'])["datatype"],
+                'target_nodata': pygeoprocessing.get_raster_info(
+                    file_registry['ndvi_base_aligned_masked'])["nodata"][0]},
+            dependent_task_list=[mask_base_ndvi_task, kernel_task],
+            target_path_list=[file_registry['ndvi_base_buffer_mean']],
+            task_name="calculate mean baseline NDVI within buffer")
+
+        mean_buffer_alt_ndvi_task = task_graph.add_task(
+            func=pygeoprocessing.convolve_2d,
+            args=(
+                (file_registry['ndvi_alt_aligned_masked'], 1),
+                (file_registry['kernel'], 1),
+                file_registry['ndvi_alt_buffer_mean']),
+            kwargs={
+                'ignore_nodata_and_edges': True,
+                'mask_nodata': True,
+                'normalize_kernel': True,
+                'target_datatype': pygeoprocessing.get_raster_info(
+                    file_registry['ndvi_alt_aligned_masked'])["datatype"],
+                'target_nodata': pygeoprocessing.get_raster_info(
+                    file_registry['ndvi_alt_aligned_masked'])["nodata"][0]},
+            dependent_task_list=[mask_alt_ndvi_task, kernel_task],
+            target_path_list=[file_registry['ndvi_alt_buffer_mean']],
+            task_name="calculate mean alternate NDVI within buffer")
+
+        # NOTE: this is the first step where the nodata value of the output
+        # raster is set based on pygeoprocessing default for the raster's
+        # datatype (rather than using the raster's native nodata)
+        delta_ndvi_task = task_graph.add_task(
+            func=pygeoprocessing.raster_map,
+            args=(lambda base_ndvi, alt_ndvi: alt_ndvi - base_ndvi,
+                  [file_registry['ndvi_base_buffer_mean'],
+                   file_registry['ndvi_alt_buffer_mean']],
+                  file_registry['delta_ndvi']),
+            target_path_list=[file_registry['delta_ndvi']],
+            dependent_task_list=[mean_buffer_base_ndvi_task,
+                                 mean_buffer_alt_ndvi_task],
+            task_name="calculate delta ndvi"  # change in nature exposure
+        )
+
+        pop_raster_info = pygeoprocessing.get_raster_info(
+            args['population_raster'])
+        pop_sr = osr.SpatialReference()
+        pop_sr.ImportFromWkt(pop_raster_info['projection_wkt'])
+        if not pop_sr.IsSame(aoi_sr):
+            transformed_bounding_box = pygeoprocessing.transform_bounding_box(
+                pop_raster_info['bounding_box'],
+                pop_raster_info['projection_wkt'],
+                aoi_projection)
+            pop_raster_info['bounding_box'] = transformed_bounding_box
```

Comment:
Is this needed or used? Or is everything being handled in `utils.resample_population_raster`? Sorry if I'm missing something here.